ui/static/ui/js/learningresources/learning_resource_panel.jsx

```diff
@@ -204,6 +228,7 @@ define("learning_resource_panel", ['react', 'jquery', 'lodash',
               message: undefined,
               description: description,
               previewUrl: previewUrl,
+              descriptionOriginal: _.cloneDeep(description),
```

Comment:
It is not necessary to use `cloneDeep` here since Javascript strings are immutable